pkg/cableengine/ipsec/strongswan.go

```diff
@@ -138,6 +153,15 @@ func (i *strongSwan) ConnectToEndpoint(endpoint types.SubmarinerEndpoint) (strin
 		Encap:       "yes",
 		Mobike:      "no",
 	}
+
+	// We point to the remote port that has proven to work by trial and error
```

Comment:
Just checking: Is this not a documented mechanism?

Reply:
Documentation is unclear about this.. nobody tells you to point to the NATT IKE port, and if you don't do that sometimes does not work... 

Also, surprisingly, if you declare the remote port, and it's the standard port (ike or not...) dataplane will stop working, it's full of bugs.